src/components/PostPageDetail/index.js

```diff
@@ -660,33 +672,36 @@ const PostPageDetailIdComponent = (props) => {
                   isLoading={loadingPost}
                   refreshComment={handleRefreshComment}
                   refreshChildComment={handleRefreshChildComment}
-                  navigateToReplyView={(data) =>
-                    navigateToReplyView(
-                      data,
-                      updateParentPost,
-                      findCommentAndUpdate,
-                      item,
-                      updateVoteLatestChildren,
-                      getComment
-                    )
-                  }
                   findCommentAndUpdate={findCommentAndUpdate}
                   contextSource={contextSource}
                   updateVote={handleUpdateVote}
                   isShortText={isShortText}
+                  onReplyButtonClick={(reactiondId, username, level) => {
+                    if (level === 0 || level === 1) {
+                      setTypeComment('child');
+                    }
+                    setReactionId(reactiondId);
+                    setReplyUsername(username);
+                  }}
                 />
               )
             )}
           </ScrollView>
 
           <WriteComment
             postId={feedId}
-            username={handleUserName(item)}
+            username={replyUsername || handleUserName(item)}
             value={textComment}
             onChangeText={(value) => setTextComment(value)}
             onPress={onComment}
             loadingPost={loadingPost}
             isKeyboardOpen={isKeyboardOpen}
+            onClear={() => {
+              setTextComment('');
+              setTypeComment('parent');
+              setReplyUsername('');
+            }}
+            isReply={typeComment === 'child' && replyUsername}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `navigateToReplyView` prop has been removed from the component being rendered, and a new `onReplyButtonClick` prop has been added. This change seems to be part of a larger refactor of the commenting functionality. Ensure that the component being rendered can handle this new prop and that all instances where `navigateToReplyView` was used have been updated.
```diff
- navigateToReplyView={(data) =>
-   navigateToReplyView(
-     data,
-     updateParentPost,
-     findCommentAndUpdate,
-     item,
-     updateVoteLatestChildren,
-     getComment
-   )
- }
+ onReplyButtonClick={(reactiondId, username, level) => {
+   if (level === 0 || level === 1) {
+     setTypeComment('child');
+   }
+   setReactionId(reactiondId);
+   setReplyUsername(username);
+ }}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->